content/fr/concepts/observability-primer.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Introduction à l'observabilité
+description: Concepts fondamentaux de l'observabilité
+weight: 9
+default_lang_commit: 71833a5f8b84110dadf1e98604b87a900724ac33
+cSpell:ignore: webshop
+---
+
+## Qu'est-ce que l'observabilité ?
+
+L'observabilité est la capacité à comprendre l'état d'un système en examinant
+ses données sortantes, sans avoir besoin de connaître son fonctionnement
+interne. Elle permet non seulement de résoudre facilement les problèmes et
+d'appréhender les nouveaux "inconnus inconnus" mais également de répondre à la
+question "Pourquoi cela arrive-t-il ?"
+
+Pour pouvoir poser ce type de questions à votre système, votre application doit
+être correctement instrumentée, c'est-à-dire que votre code doit émettre :
+[des signaux](/docs/concepts/signals/) tels que
+[des traces](/docs/concepts/signals/traces/),
+[des métriques](/docs/concepts/signals/metrics/), et
+[des logs](/docs/concepts/signals/logs/).
+
+Une application est correctement instrumentée si les développeurs disposent de
+toutes les informations nécessaires pour corriger un problème et n'ont pas
+besoin d'ajouter une instrumentation supplémentaire.
+
+[OpenTelemetry](/docs/what-is-opentelemetry/) est le mécanisme permettant
+d'instrumenter le code des applications afin de rendre le système observable.
+
+## Fiabilité et métriques
+
+Le terme **télémétrie** fait référence aux données émises par un système et son
+comportement. Les données peuvent prendre la forme de
+[traces](/docs/concepts/signals/traces/), de
+[métriques](/docs/concepts/signals/metrics/), et de
+[logs](/docs/concepts/signals/logs/).
+
+La **fiabilité** répond à la question : "Est-ce que le service fait ce que les
+utilisateurs attendent de lui ?" Un système peut afficher un pourcentage de
+disponibilité de 100%, mais s'il ne répond pas à la demande de l'utilisateur
+(par exemple : ajouter une paire de chaussettes noires dans le panier à chaque
+fois qu'il clique sur "Ajouter au panier"), alors le système pourrait être
+considéré comme **peu** fiable.
+
+Les **métriques** sont un ensemble de données numériques collectées pour votre
+infrastructure ou votre application sur une période donnée. Le nombre d'erreurs
+système, le nombre d'erreurs sur les requêtes ainsi que l'utilisation mémoire
+d'un service donné sont quelques exemples de métriques. Pour plus d'informations
+sur les métriques et leur rôle dans OpenTelemetry, référez-vous à la page
+[Métriques](/docs/concepts/signals/metrics/).
+
+L'indicateur de niveau de service, également connu sous le nom de **SLI**, est
+un indicateur de fonctionnement d'un service qui est évalué côté utilisateur. La
+vitesse à laquelle une page Web se charge est un exemple de SLI.
+
+Les objectifs de niveau de service, communément appelé **SLO**, permettent de
```

Comment:
Les objectifs ... appelés (au pluriel) ?

Reply:
Thanks @marcalff for pointing this out. The mispelling is now fixed.